backend/app/utils/credentials/__init__.py

```
@@ -1 +1,9 @@
-from .smartapi_credentials import *
+from .base_credential_manager import CredentialManager
```

Pylint (3.3.7)

[error] Cannot import 'base_credential_manager' due to 'invalid syntax (credentials.base_credential_manager, line 13)'

(E0001)